container.te

```diff
@@ -1405,12 +1405,19 @@ fs_mounton_cgroup(container_engine_t)
 fs_unmount_cgroup(container_engine_t)
 fs_manage_cgroup_dirs(container_engine_t)
 fs_manage_cgroup_files(container_engine_t)
-fs_mount_tmpfs(container_engine_t)
 fs_write_cgroup_files(container_engine_t)
-
-allow container_engine_t proc_t:file mounton;
-allow container_engine_t sysctl_t:file mounton;
-allow container_engine_t sysfs_t:filesystem remount;
+fs_remount_cgroup(container_engine_t)
+fs_mount_all_fs(container_engine_t)
+fs_remount_all_fs(container_engine_t)
+fs_unmount_all_fs(container_engine_t)
+
+allow container_engine_t container_file_t:chr_file mounton;
+allow container_engine_t devpts_t:chr_file { open setattr };
+allow container_engine_t filesystem_type:{dir file} mounton;
+allow container_engine_t proc_kcore_t:file mounton;
+allow container_engine_t proc_t:{dir file} mounton;
+allow container_engine_t sysctl_irq_t:dir mounton;
```

Comment:
kernel_mounton_all_sysctls(container_engine_t)

Reply:
where's a good place to find all of these macros?

Reply:
/usr/share/selinux-policy/devel/include/*

Reply:
But audit2allow -R will find some for you.